multithreading.c

```diff
@@ -25,7 +25,7 @@ void *thr_func(void *arg) {
   thread_data_t *data = (thread_data_t *)arg;
 
  // making sure that every thread gets a different set of random numbers
-  int seed = time(NULL)+data->tid;
+  int seed = time(NULL) + (1 + data->tid);
```

Comment:
I've already corrected the multiplying by zero error in a previous commit (here https://github.com/5N44P/computational-madness/commit/af902d360f63b6c459a657d9d9eed8db274f8158) by chainging it to a sum